assets/js/components/ChecksCatalog/ChecksCatalogNew.jsx

```diff
@@ -17,19 +18,12 @@ export const ChecksCatalogNew = () => {
   } = useSelector(getCatalog());
 
   useEffect(() => {
-    dispatchUpdateCatalog();
+    dispatchUpdateCatalog()(dispatch);
```

Comment:
As mentioned elsewhere could this simply be `dispatchUpdateCatalog(dispatch);`?

And also other similar usages.